publish.sh

```diff
@@ -0,0 +1,2 @@
+#! /bin/sh
```

Comment:
Rename this file to something understandable. (Applies to the other projects as well)